core/storage/email/gae_models.py

```diff
@@ -46,7 +46,7 @@ class SentEmailModel(base_models.BaseModel):
     recipient_email = ndb.StringProperty(required=True)
     # The user ID of the email sender. For site-generated emails this is equal
     # to SYSTEM_COMMITTER_ID.
-    sender_id = ndb.StringProperty(required=True)
+    sender_id = ndb.StringProperty(required=True, indexed=True)
```

Comment:
Ditto

Reply:
Yes, we will need migration.

Reply:
Removing `SentEmailModel` and `BulkEmailModel` from this PR from now.